tests/Bridge/Laravel/Providers/HealthServiceProviderTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\EoneoPay\Externals\Bridge\Laravel\Providers;
+
+use EoneoPay\Externals\Bridge\Laravel\Providers\HealthServiceProvider;
+use EoneoPay\Externals\Health\Health;
+use EoneoPay\Externals\Health\Interfaces\HealthInterface;
+use Tests\EoneoPay\Externals\Stubs\Vendor\Illuminate\Contracts\Foundation\ApplicationStub;
+use Tests\EoneoPay\Externals\TestCase;
+
+/**
+ * @covers \EoneoPay\Externals\Bridge\Laravel\Providers\HealthServiceProvider
+ */
+class HealthServiceProviderTest extends TestCase
+{
+    /**
+     * Test that service provider registers the health service into the Laravel application.
+     *
+     * @return void
+     */
+    public function testRegister(): void
+    {
+        $application = new ApplicationStub();
```

Comment:
It'd be worth tagging a health check stub here and asserting that it got injected into the constructed Health service